feat: Add documentation about Apache Airflow (#1880)
getsentry/sentry-docs: src/collections/_documentation/platforms/python/airflow.md

---
title: Apache Airflow
sidebar_order: 9
---

> [Apache Airflow](https://airflow.apache.org/) **1.10.6** and above can be setup to send errors to Sentry.

Reviewer: ```suggestion
[Apache Airflow](https://airflow.apache.org/) **1.10.6** and above can be set up to send errors to Sentry.
```

## Installation

Install the `apache-airflow` package with the `sentry` requirement.

```py
pip install 'apache-airflow[sentry]'
```

> You can then add your Sentry DSN to your configuration file (ex. `airflow.cfg`) under the `[sentry]` field.

Reviewer: ```suggestion
Then, add your Sentry DSN to your configuration file (ex. `airflow.cfg`) under the `[sentry]` field.
```

```ini
[sentry]
sentry_dsn = ___PUBLIC_DSN___
```

> Now, Airflow should report errors to Sentry automatically. Airflow will also generate custom tags and breadcrumbs based on the current dag and tasks at time of error.

Reviewer: ```suggestion
Now, Airflow should report errors to Sentry automatically. Airflow will also generate custom tags and breadcrumbs based on the current dag and tasks at time of error.
```
```suggestion
Now, Airflow should report errors to Sentry automatically. Airflow will also generate custom tags and breadcrumbs based on the current dag and tasks at the time of the error.
```

Reviewer: What's a dag? 😄

Author: A dag is a direct acyclic graph - https://en.wikipedia.org/wiki/Directed_acyclic_graph

> In Airflow, a DAG – or a Directed Acyclic Graph – is a collection of all the tasks you want to run, organized in a way that reflects their relationships and dependencies.
https://airflow.apache.org/docs/stable/concepts.html#dags

It seems in Airflow docs and code they choose to capitalize `DAG`, do you think that we should do the same here?

Reviewer: Yup. 😸 I updated the suggested edit. When using an acronym, write out the words explicitly the first time you mention it with the acronym in parentheses. After that initial definition, feel free to use just the acronym.

Author: Cool thank you, will keep that in mind.
